### DISCUSSION TITLE
Android ReadOnly TextBox and Text Selection

### DISCUSSION THREAD

**Discussion by ajpinedam** [unoplatform/uno#10473] [General]
Hey @jeanplevesque 

We are working on the issue https://github.com/unoplatform/uno/issues/9908.

We got the functionality to disallow the user from modifying the text of a ReadOnly TextBox while being able to select the text and copy it.

There is a behaviour we see, though, that the keyboard shows up every time we make a new selection.

Is this something your project can live with?

I will attach a video in the comment so you can see it.

**Comment by ajpinedam** [comment]
https://user-images.githubusercontent.com/1900897/202560354-680bacf1-89cf-4b86-a725-fd393e67ae1a.mp4

**Reply by jeanplevesque** [reply]
It looks a bit weird with the keyboard bouncing, but it's good enough if we can select text in order to copy it, which is the important part. 😊
